cpp/basix/cell.cpp

```diff
@@ -15,7 +15,7 @@ xt::xtensor<double, 2> cell::geometry(cell::type celltype)
   switch (celltype)
   {
   case cell::type::point:
-    return xt::xtensor<double, 2>({{0.0}});
+    return xt::xtensor<double, 2>({{}});
```

Comment:
Is this change intended?

Reply:
Maybe.

I think the shape of `cell::geometry` should be npoints by tdim, so in this case 1 by 0.

On the other hand, this isn't the change that fixed the space on points, so we could alternatively have the shape as npoints by gdim and declare that the gdim for a point is 1.